src/plugin/regions.js

```diff
@@ -230,13 +230,14 @@ class Region {
     bindInOut() {
         this.firedIn = false;
         this.firedOut = false;
-
+        const threshold = 5e-2;
```

Comment:
@huanle0610 can you add a comment for the `treshold` var, explaining why it's that specific value?